resources/frameworks.yaml

```diff
@@ -59,11 +59,10 @@ AutoGluon_best:
     presets: best_quality
 
 autosklearn:
-  version: '0.8.0'
+  version: '0.11.1'
   project: https://automl.github.io/auto-sklearn/
-#  params:
-#    _save_artifacts: ['models']
-#    _n_jobs: 1
+  params:
+    _save_artifacts: ['debug_as_files']
```

Comment:
I'm fine with having 2 params for the debug format, but can we disable them by default?

Reply:
Yes, sure. Just pushed this change.